src/Backend/Facebook.php

```diff
@@ -42,11 +43,10 @@ public function getRequest($url)
      */
     public function extractCount(array $data)
     {
-        if (isset($data['data']) && isset($data['data'][0]) && isset($data['data'][0]['total_count'])) {
-            return $data['data'][0]['total_count'];
-        }
-        if (isset($data['share']) && isset($data['share']['share_count'])) {
-            return $data['share']['share_count'];
+        if (isset($data['engagement']) && isset($data['engagement']['reaction_count'])
```

Comment:
`(isset($data['engagement'])` is not needed when having all those other checks

Reply:
it was like that in the present code, too:
if (isset($data['share']) && isset($data['share']['share_count'])) { ...

Reply:
do you know why travis fails?

Reply:
https://travis-ci.org/heiseonline/shariff-backend-php/jobs/297185279#L601

> Failed to clone the git@github.com:heiseonline/shariff-backend-php.git repository

Seems to hit some github limitation. For sure not your fault.

> it was like that in the present code, too

Does not mean that it was "correct" like that. ;-)
Simplifying code makes it better readable.